### DISCUSSION TITLE
Skip deploys for commits that have [skip-ci] in the name

### DISCUSSION THREAD

**Discussion by paales** [vercel/vercel#5765] [Help] [answered]
We have commits like this:

<img width="1202" alt="Schermafbeelding 2021-01-28 om 15 54 50" src="https://user-images.githubusercontent.com/1244416/106155626-2c7d0b80-6181-11eb-9932-c9abbd27244c.png">

After a new commit to master is made (usually because a PR is merged) our semantic-release-bot will create a new release in the repository, this updates the version in the package.json.

The result of this that effectively every deployment is done twice, is there a way to skip these deployments?

**Comment by styfle** [comment]
Yes, take a look at the answer in this discussion: https://github.com/vercel/vercel/discussions/4008#discussioncomment-114386

**Comment by paales** [comment, marked as answer]
To only deploy the released version or a random branch:

```bash
bash -c '[[ $VERCEL_GIT_COMMIT_REF == "master" ]] && echo $VERCEL_GIT_COMMIT_MESSAGE | grep -v "chore: release"'
```

**Reply by joaoguilhermee** [reply]
Where do I need to run this code?

**Reply by amyegan** [reply]
It goes in the ["Ignored Build Step" field](https://vercel.com/guides/how-do-i-use-the-ignored-build-step-field-on-vercel) in your project settings
